Source/WebCore/Modules/mediasource/SourceBuffer.cpp

```diff
@@ -594,7 +594,7 @@ void SourceBuffer::sourceBufferPrivateAppendComplete(AppendResult result)
     monitorBufferingRate();
     m_private->reenqueueMediaIfNeeded(m_source->currentTime());
 
-    DEBUG_LOG(LOGIDENTIFIER, "buffered = ", m_private->buffered());
+    ALWAYS_LOG(LOGIDENTIFIER, "buffered = ", m_private->buffered(), ", totalBufferSize: ", m_private->totalTrackBufferSizeInBytes());
```

Comment:
Do we want to log this every time?

Reply:
I think so, yes. It's only logged when appends happen, it's not per-sample, and it has super useful info for debugging purposes.